controller/search_blackbox_test.go

```diff
@@ -930,3 +930,83 @@ func (s *searchBlackBoxTest) TestIncludedParents() {
 	}
 	assert.Equal(s.T(), successCnt, 4)
 }
+
+func (s *searchBlackBoxTest) TestFilterAssigneeNullAfterWIUpdate() {
+	identitiesTestFixtures := tf.NewTestFixture(s.T(), s.DB, tf.Iterations(1), tf.Areas(1), tf.WorkItems(1))
```

Comment:
Please use `fxt` instead `identitiesFxt`. We use `fxt` almost everywhere else.

Reply:
Done. 9c745b299aeb812f39f5a60850d5334b4c973153

Reply:
Why don't you use `CreateWorkItemEnvironment()` function instead of `tf.NewTestFixture(s.T(), s.DB, tf.Iterations(1), tf.Areas(1), tf.WorkItems(1))`. Checkout this -> https://github.com/fabric8-services/fabric8-wit/pull/1682/files

Reply:
Done. c6dbc774a542db616d1e27dc4c277ca529d7e481